eralchemy2/version.py

```diff
@@ -1 +1,3 @@
-version = '1.3.3'
+from __future__ import annotations
+
+version = "1.3.3"
```

Comment:
is this needed?
i'd like to have the version as a separate file, as this will otherwise break the `make_release.py` script in the script folder

Reply:
File was deleted

Reply:
yes I resolved this by changing the release process to poetry :laughing: